src/main/java/practice/StreamPractice.java

```diff
@@ -23,7 +32,11 @@ public int findMinEvenNumber(List<String> numbers) {
      * But before that subtract 1 from each element on an odd position (having the odd index).
      */
     public Double getOddNumsAverage(List<Integer> numbers) {
-        return 0D;
+        return IntStream.range(0, numbers.size())
+                .map(n -> n % 2 == 1 ? numbers.get(n) - 1 : numbers.get(n))
```

Comment:
you can simplify here
```suggestion
                .map(i -> numbers.get(i) - i % 2)
```